eggo/config.py

```diff
@@ -18,7 +18,6 @@
 
 from eggo.util import random_id
 
-
```

Comment:
nit: I think pep8 will yell about this, as it likes 2 lines between top-level items